cmd/virt-launcher/virt-launcher.go

```diff
@@ -238,6 +239,12 @@ func main() {
 
 	log.InitializeLogging("virt-launcher")
 
+	// Block until all requested hookSidecars are ready
```

Comment:
why we need to wait here for the sidecars? 
let say our vm is running a database and our sidecar need to connect to our database to be ready it will stuck in a loop.

Reply:
hookside car does not connect to anything. virt-launcher connects to hook sidecar and must do this before the initialization process begins.

Reply:
Now I understand the usage Thanks.